docs/contributing.md

```diff
@@ -42,7 +42,10 @@ Clone this repository `git clone git@github.com:codeready-toolchain/toolchain-e2
 
 This repository provides you multiple Makefile targets that you can use - it depends on which version of KubeSaw operators you want to install.
 
-**NOTE**: If the cluster is an OSD cluster, then set the variable `IS_OSD=true` when running any of the Makefile targets (for example: `make dev-deploy-latest IS_OSD=true`).
+NOTE:
+If the connection to the cluster requires you to manually set the `--insecure-skip-tls-verify=true`,
+you can do so when running any of the Makefile targets by setting the DISABLE_KUBE_CLIENT_TLS_VERIFY variable to true
+(for example: `make appstudio-dev-deploy-latest DISABLE_KUBE_CLIENT_TLS_VERIFY=true`).
```

Comment:
```suggestion
```suggestion
NOTE:
Set the `DISABLE_KUBE_CLIENT_TLS_VERIFY` environment variable to `true` when the TLS connection to the cluster cannot be verified. For example: 
```

Reply:
thanks for the suggestion @xcoulon , looks good to me. But unless I'm seeing it wrong, the example is missing .

Reply:
Ah damn'it, I had issues with the UI when providing the suggestion on the lines 45 to 48, made some changes and lost the example along the way 🤦‍♂️ 
Let me fix that!